File: Python/Which Alien.py
# 2011 J1

antenna = int(input("How many antennas?\n"))
eyes = int(input("How many eyes?\n"))

if antenna >= 3 and eyes <= 4:
    print("TroyMartian")

if antenna <= 6 and eyes >= 2:
    print("VladSaturnian")

if antenna <= 2 and eyes <= 3:
    print('GraemeMercurian')

File: C++/2011/WhichAlien.cpp
// J1

#include <iostream>

using namespace std;

int main()
{
    cout << "How many antennas?" << endl;
    int antennas;
    cin >> antennas;

    cout << "How many eyes?" << endl;
    int eyes;
    cin >> eyes;

    if (antennas >= 3 && eyes <= 4)
        cout << "TroyMartian" << endl;

    if (antennas <= 6 && eyes >= 2)
        cout << "VladSaturnian" << endl;

    if (antennas <= 2 && eyes <= 3)
        cout << "GraemeMercurian" << endl;
}